prow/cmd/tackle/main.go

```diff
@@ -46,6 +46,62 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" // for gcp auth provider
 )
 
+// printArray: Print items in collection (up to a non-zero limit); return a bool indicating if results were truncated.
```

Comment:
fyi the general way people writes this is like a sentence: `printArray prints the collection items (up to a non-zero limit)`

Reply:
ref: https://golang.org/doc/effective_go.html#commentary